### DISCUSSION TITLE
申请退款，购买一年专业版，激活一周后变为免费版，一直未解决

### DISCUSSION THREAD

**Discussion by LuLu120325** [getlantern/lantern-forum-cn#910] [General]
邮箱：[2647611833@qq.com](mailto:2647611833@qq.com)
这是付款凭证
![IMG_7493](https://github.com/user-attachments/assets/7aa5551a-4a26-413e-815b-eb140d3bf51c)

**Comment by MaxMKMax** [comment]
@LuLu120325 您好. 您先别退款好不好? 我让他们给您补偿. 加长时间. 我帮您提交问题上去让他们尽快解决.

**Reply by LuLu120325** [reply]
请帮我退款吧，谢谢，lantern是不错的工具，等稳定了我还是很乐意用的

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢，邮箱：2647611833@qq.com

**Reply by wwei20240912** [reply]
有问题的都跟支付宝投诉这家收款公司支付宝账号“广州饶铭网络科技有限公司”，要求协助退款。

**Comment by MaxMKMax** [comment]
@LuLu120325 好的. 既然您这么坚决, 会帮您提交退款申请.

**Reply by wwei20240912** [reply]
@Mike777Khoo 我的遇到的也是同样问题，请帮忙处理退款问题，一直收不到验证码无法使用专业版本，邮箱：94472127@qq.com

**Reply by LuLu120325** [reply]
好的，会继续跟踪退款后续，谢谢你们

**Comment by wwei20240912** [comment]
您好，购买的专业版完全无法使用，收不到验证码，请帮忙协助退款，购买凭证如下：


------------------ 原始邮件 ------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月8日(星期二) 晚上11:10
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 申请退款，购买一年专业版，激活一周后变为免费版，一直未解决 (Discussion #910)





 
@LuLu120325 好的. 既然您这么坚决, 会帮您提交退款申请.
